Vagrantfile.dist

```diff
@@ -57,3 +57,9 @@ class CustomConfig
 end
 
 load 'virtualization/rawbot-virtualization/Vagrantfile'
+
+Vagrant.configure("2") do |config|
+   if ENV['GITLAB_CI'] && ENV['DO_GLOBAL_PROJECTS_CACHE']
+        config.vm.synced_folder "/home/gitlab-runner/projects_cache/", "/home/vagrant/.projects_cache"
+   end
+end
```

Comment:
I would say this has its place in the "global"

Or is it something that is subject to change often ?